src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -285,6 +293,35 @@ class TestRequestRateManager : public TestLoadManagerBase,
     CheckSequences(params_.num_of_sequences);
   }
 
+  void TestSequenceBalance()
```

Comment:
Testing is a place where we can/should add comments. For something like this, looking at it it is not clear what the purpose of this function is.

Also, all of our functions that start with `Test` generally drive some stimulus and then confirm behavior. If this function is only confirming the result then the function should start with `Check` instead of `Test`